store/views.py

```diff
@@ -77,6 +124,43 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    if request.method=="POST":
+        try:
+            book_id=request.POST['bid']
+            book=BookCopy.objects.get(pk=book_id)
+            book.status=True
+            book.borrower=None
+            book.borrow_date=None
+            book.save()
+
+            return JsonResponse({'message':"book successfull return"})
+
+        except:
+            return JsonResponse({'message':"Try again"})
+
+
+    return JsonResponse({'message':"BAD BOY"})
+
+@csrf_exempt
+@login_required 
+def rateBookView(request):
+    if request.method=="POST":
+        star=request.POST["stars"]
+        book_id=request.POST["bid"]
+        current_user=request.user
+        rating=Rating.objects.filter(book__exact=BookCopy.objects.get(id=book_id).book,user__exact=current_user)
+        if rating:
+            rating01=rating[0]
+            rating01.stars=star
+            rating01.save()
+            return JsonResponse({"message":"rating has been updated"})
+        else:
+            rating01=Rating.objects.create(user=current_user,book=BookCopy.objects.get(id=book_id).book,stars=star)
```

Comment:
This is not a good way to call ORM queries. This will run two queries on database.
Better, use `book=book_id`.